### ISSUE TITLE
Cookiecutter treeviewitem Names are not accessible to Screen Readers

### ISSUE THREAD

**Issue by jsschultz** [Microsoft/PTVS#1951]
The cookiecutter treeview does not properly define AutomationProperties for its items, so Windows UIAutomation api exposes the type name of the viewmodels backing the items. This prevents users relying on assistive technology (such as screen readers) from knowing what those items are.

**Comment by huguesv** [comment]
Thanks for the report. We'll have a look.
